go/chat/storage/inbox.go

```diff
@@ -526,10 +526,11 @@ func (i *Inbox) applyQuery(ctx context.Context, query *chat1.GetInboxQuery, rcs
 			*query.TlfVisibility != conv.Metadata.Visibility {
 			ok = false
 		}
-		if query.UnreadOnly && conv.ReaderInfo.ReadMsgid >= conv.ReaderInfo.MaxMsgid {
+		isUnread := conv.IsUnread()
```

Comment:
We should only call this if the query param is set.

Reply:
done